server/src/main/java/io/crate/metadata/SimpleReference.java

```diff
@@ -133,11 +138,13 @@ public SimpleReference(ReferenceIdent ident,
         this.hasDocValues = hasDocValues;
         this.defaultExpression = defaultExpression;
         this.oid = oid;
+        this.isDropped = isDropped;
     }
 
     /**
      * Returns a cloned Reference with the given ident
      */
+    @Override
```

Comment:
We need to reflect new flag in `toMapping` - I think later on DROP COLUMn will be updating  (actual cloning with new flag since refs are immutable) Reference in a new transport action

Reply:
Thx, added it.